### PR TITLE
Theme some hard to see faces in solarized light

### PR DESCRIPTION
**Pull request by r-darwish**

### CODE REVIEW

**Comment by ema2159** [comment]
Please show a before and after screenshot so it's easier to review!

**Comment by r-darwish** [comment]
image

File: themes/doom-solarized-light-theme.el [hunk 1]
```diff
@@ -120,6 +120,8 @@ determine the exact padding."
    (org-block-background :background (doom-blend yellow bg 0.04))
    (org-block-begin-line :background (doom-blend yellow bg 0.08))
    (org-block-end-line :background (doom-blend yellow bg 0.08))
+   (lsp-ui-sideline-code-action :foreground (doom-color 'blue))
```
Comment (hlissner) [inline themes/doom-solarized-light-theme.el]: Using `doom-color` here is unnecessary.

```diff
-   (lsp-ui-sideline-code-action :foreground (doom-color 'blue))
+   (lsp-ui-sideline-code-action :foreground blue)
```

**Comment by hlissner** [comment]
Looks good to me. Thanks for the help!

**Pull request merged**